sklearn/metrics/_plot/tests/test_roc_curve_display.py

```diff
@@ -302,7 +304,7 @@ def test_plot_roc_curve_pos_label(pyplot, response_method, constructor_name):
     classifier = LogisticRegression()
```

Comment:
I thought about adding `from_cv_results` to this (`test_plot_roc_curve_pos_label`) test BUT I see we create an imbalanced training set, and pass a separate test set to `from_predictions` and `from_estimator`, which we could not do with `from_cv_results`, so it feels like it would miss the spirit of this test. I think we should add a separate test for `pos_label` `from_cv_results`, WDYT?